tournamentcontrol/competition/sites.py

```diff
@@ -1237,6 +1249,42 @@ def forfeit(self, request, competition, season, extra_context, match, **kwargs):
         templates = self.template_path("forfeit.html")
         return self.render(request, templates, context)
 
+    @competition_by_slug_m
+    def stream_instructions(
+        self, request, competition, season, format, extra_context, **kwargs
+    ):
+        """
+        Render live streaming instructions template with season-specific data.
+        Supports both markdown (.md) and HTML (.html) output formats.
+        """
+        if extra_context is None:
+            extra_context = {}
+
+        # Find all grounds with stream keys using ORM reverse relations
+        streaming_grounds = Ground.objects.filter(
+            venue__season=season, stream_key__isnull=False
+        ).exclude(stream_key="")
+
+        context = {
+            "competition": competition,
+            "season": season,
+            "streaming_grounds": streaming_grounds,
+        }
+        context.update(extra_context)
+
+        # Render the markdown template
+        template_path = "tournamentcontrol/competition/live_streaming_instructions.md"
+        content = render_to_string(template_path, context, request=request)
```

Comment:
Use convention for finding templates with `self.template_path` method. Search rest of file for examples.